CppCoreGuidelines.md

```diff
@@ -9952,7 +9952,7 @@ Arithmetic rules:
 * [ES.103: Don't overflow](#Res-overflow)
 * [ES.104: Don't underflow](#Res-underflow)
 * [ES.105: Don't divide by zero](#Res-zero)
-* [ES.106: Don't try to avoid negative values by using `unsigned`](#Res-nonnegative)
+* [ES.106: Don't try to avoid negative values by using `unsigned`](#Res-non-negative)
```

Comment:
this changes an anchor, which may break external links

Reply:
Thank you for the feedback!
This anchor will work properly because I changed it in both side

Reply:
I am referring to external links. For example, MSVC compiler warning C26494 links to #Pro-type-init. I don't know if anyone links to #Res-nonnegative, but I wouldn't be surprised.

Reply:
Oh, I see,
Thank you for the information!

Reply:
I modified three links like below and sadly MSVC has external links with `Res-nonnegative`
I will let them know or make a PR for their documentation [here](https://github.com/MicrosoftDocs/cpp-docs) if this PR is accepted

* Res-nonnegative
* Rt-nondependent
* Rt-nongeneric

Reply:
I resolve this conversation
If you have any suggestion please let me know !